### ISSUE TITLE
Vagrant cannot connect in ssh

### ISSUE THREAD

**Issue by cbornet** [jhipster/jhipster-devbox#41]
After vagrant init, the default user to connect to the box will be "vagrant" but it doesn't exist on the box
Without ssh, most vagrant commands will not work and you can't mount the shared directories.

Two solutions:
* after vagrant init, set in Vagrantfile
```
config.ssh.username = "jhipster"
config.ssh.password = "jhipster"
```
* create the user vagrant during provisioning
```
useradd vagrant --password wthgG9Dxlk13E --home /home/vagrant --create-home -s /bin/bash
echo "vagrant ALL=(ALL) NOPASSWD:ALL" > /etc/sudoers.d/vagrant
mkdir -p /home/vagrant/.ssh
wget --no-check-certificate https://raw.github.com/mitchellh/vagrant/master/keys/vagrant.pub -O /home/vagrant/.ssh/authorized_keys
chmod 0700 /home/vagrant/.ssh
chmod 0600 /home/vagrant/.ssh/authorized_keys
chown -R vagrant /home/vagrant/.ssh
```
and config.ssh.insert_key = false (otherwise the vagrant will try to use the ubuntu private_key)

**Comment by jdubois** [comment]
I'll try the first solution as it looks easier, and won't fail when they add a Vagrant user

**Comment by cbornet** [comment]
And jhipster must be a passwordless sudoer of course !

**Comment by jdubois** [comment]
Closed as we will switch back to Trusty - see #43
